src/_imaging.c

```diff
@@ -94,40 +94,38 @@
 #include <math.h>
 
 /* Configuration stuff. Feel free to undef things you don't need. */
-#define WITH_IMAGECHOPS /* ImageChops support */
-#define WITH_IMAGEDRAW /* ImageDraw support */
-#define WITH_MAPPING /* use memory mapping to read some file formats */
-#define WITH_IMAGEPATH /* ImagePath stuff */
-#define WITH_ARROW /* arrow graphics stuff (experimental) */
-#define WITH_EFFECTS /* special effects */
-#define WITH_QUANTIZE /* quantization support */
-#define WITH_RANKFILTER /* rank filter */
-#define WITH_MODEFILTER /* mode filter */
-#define WITH_THREADING /* "friendly" threading support */
+#define WITH_IMAGECHOPS  /* ImageChops support */
+#define WITH_IMAGEDRAW   /* ImageDraw support */
+#define WITH_MAPPING     /* use memory mapping to read some file formats */
+#define WITH_IMAGEPATH   /* ImagePath stuff */
+#define WITH_ARROW       /* arrow graphics stuff (experimental) */
+#define WITH_EFFECTS     /* special effects */
+#define WITH_QUANTIZE    /* quantization support */
+#define WITH_RANKFILTER  /* rank filter */
+#define WITH_MODEFILTER  /* mode filter */
+#define WITH_THREADING   /* "friendly" threading support */
 #define WITH_UNSHARPMASK /* Kevin Cazabon's unsharpmask module */
 
-#undef    VERBOSE
+#undef VERBOSE
 
-#define B16(p, i) ((((int)p[(i)]) << 8) + p[(i)+1])
-#define L16(p, i) ((((int)p[(i)+1]) << 8) + p[(i)])
-#define S16(v) ((v) < 32768 ? (v) : ((v) - 65536))
+#define B16(p, i) ((((int)p[(i)]) << 8) + p[(i) + 1])
+#define L16(p, i) ((((int)p[(i) + 1]) << 8) + p[(i)])
+#define S16(v) ((v) < 32768 ? (v) : ((v)-65536))
 
 /* -------------------------------------------------------------------- */
 /* OBJECT ADMINISTRATION                        */
 /* -------------------------------------------------------------------- */
 
 typedef struct {
-    PyObject_HEAD
-    Imaging image;
+    PyObject_HEAD Imaging image;
```

Comment:
This doesn't look right. Perhaps the following should get formatted properly?
```suggestion
    PyObject_HEAD;
    Imaging image;
```
This change can be done in all files with a simple grep.

Reply:
Adding in the semicolon causes syntax errors on Windows (AppVeyor and GHA).

Reply:
Ah great, leave it to MSVC to apply arbitrary restrictions from a standard it doesn't even fully support yet.

After a bit of effort, I have found the following hack which works with both MSVC and GCC 4.1 and has no effect on structure size (see https://godbolt.org/z/76W7f4):
```suggestion
    PyObject_HEAD int:0;
    Imaging image;
```
The extra code can be hidden in a define such as `#define PILLOW_HEAD PyObject_HEAD int:0` and used similarly to my original comment. However, I'm not sure whether it is worth the extra effort.

Reply:
I think we should leave this if possible, PyObject_HEAD is the standard way to do it, and hiding it in a macro is additional complication that doesn’t make sense except for formatting.